apps/web/lib/mock-data/mock-join-us-section.tsx

```
@@ -0,0 +1,74 @@
+import { motion } from "framer-motion";
+import { ArrowUpRight, Megaphone, RefreshCw } from "lucide-react";
+import { type Feature } from "~/lib/types/home.types";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)